modules/emotion.yml

```diff
@@ -1,17 +1,20 @@
 name: emotion
 description: The Next Generation of CSS-in-JS for Nuxt.js
-repo: nuxt-community/emotion-module
+repo: nuxt-modules/emotion
 npm: '@nuxtjs/emotion'
 icon: emotion.png
-github: https://github.com/nuxt-community/emotion-module
-website: https://github.com/nuxt-community/emotion-module
+github: https://github.com/nuxt-modules/emotion
+website: https://github.com/nuxt-modules/emotion
 learn_more: https://emotion.sh/
 category: Extensions
 type: community
 maintainers:
   - name: Pooya Parsa
     github: pi0
     twitter: _pi0_
+  - name: Jonathan Bakebwa
+    github: codebender828
+    twitter: codebender828
 compatibility:
-  nuxt: ^2.0.0
+  nuxt: ^3.0.0
```

Comment:
Maybe `^2.0.0 || ^3.0.0` since the old version might still work?

Reply:
Oh yeah, good idea. The `@nuxtjs/emotion@0.1.0` works for v2.x. Should I go ahead and set this version?